build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/operator-framework/ansible-operator:v0.12.0
+FROM quay.io/operator-framework/ansible-operator:v0.14.1
```

Comment:
Can we use the newer command line tools now too?

Reply:
We can already use those. The adjustments were made in a previous setup (if any). I've been using `operator-sdk 0.15.2` without issue on both our Operators.